tests/integration/authentication/test_authentication.py

```diff
@@ -31,6 +31,30 @@ def test_should_return_401_if_you_login_with_invalid_password(
     assert response.status_code == status.HTTP_401_UNAUTHORIZED
 
 
+@pytest.mark.parametrize("endpoint", [TOKEN_URL, LOGIN_URL])
+def test_should_return_403_if_user_not_active(
+        endpoint: str,
+        client: TestClient,
+) -> None:
+    response = client.post(
+        endpoint, data={"username": "dod", "password": "secret"}
+    )
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+    assert "User account is not active" in response.json()["message"]
+
+
+@pytest.mark.parametrize("endpoint", [TOKEN_URL, LOGIN_URL])
+def test_should_return_403_if_user_not_verified(
+        endpoint: str,
+        client: TestClient,
+) -> None:
+    response = client.post(
+        endpoint, data={"username": "warner", "password": "secret"}
```

Comment:
You are testing that an inactive or non-verified

* cannot login
* cannot request a token.

But from what I can see, you are *not* testing that they

* cannot perform a restricted action if they are logged in or have a token already.